tools/tpm2_import.c

```diff
@@ -83,6 +84,10 @@ struct tpm_import_ctx {
     TPM2B_DATA enc_sensitive_key;
     TPM2B_MAX_BUFFER encrypted_inner_integrity;
     TPM2B_MAX_BUFFER encrypted_duplicate_sensitive;
+    UINT32 objectAttributes;
```

Comment:
Why do you need this variable? It isn't used AFAICT.

Reply:
It is used to retrieve the value of -A option from commandline:

```
 +    case 'A':
 +        if(!tpm2_util_string_to_uint32(value, &ctx.objectAttributes)) {
 +            LOG_ERR("Invalid object attribute, got\"%s\"", value);
 +            return false;
 +        }
 +        ctx.flags.A = 1;
 +        break;
```

Reply:
ah, sorry. I misread and thought you were adding both a struct tpm_import_ctx member and a global variable.